Flo's February updates (#852)
akeneo/pim-helpcenter: content/updates/2021-01/2-Manage-your-spell-check-dictionaries.md

---
pim_announcement_alt_img: "The spell-check dictionaries management feature"
pim_announcement_audience:
- EE
---

> # Manage your spell-check dictionaries

Reviewer: @camillebourjade can you validate the "spell-check dictionaries" wording?
I also saw “Spell Checking Dictionaries” or “Spell dictionaries” on Google.

Reviewer 2: spell-check dictionaries is good. Other forms exist but this one is the most commonly used :-)

::: meta-data type="New" features="Quality, Productivity" available="late January" link-to-doc="../articles/TODO"

> You can now entirely manage your spell-check dictionaries on all your locales. Not only do you gain visibility on the words you've already added, but you can also easily modify, delete or even add other words for each dictionary. A new step toward the next level in terms of enrichment productivity!

Reviewer: ```suggestion
You can now fully manage your spell-check dictionaries in all your locales. Not only will you gain visibility on the words you have already added, but you will also be able to easily edit, delete or even add new words for each dictionary. Another step towards the next level in terms of enrichment productivity!
```
Only rewording, the message is perfect!

> You can now fully manage your spell-check dictionaries in all your locales. Not only will you gain visibility on the words you have already added, but you will also be able to easily edit, delete or even add new words for each dictionary. Another step towards the next level in terms of enrichment productivity!

Author: ```suggestion
You can now fully manage your spell-check dictionaries in all your locales. Not only you gain visibility on the words you have already added, but you are also able to easily edit, delete or even add new words for each dictionary. Another step towards the next level in terms of enrichment productivity!
```

![The spell-check dictionaries management feature](../img/TODO.png)

> This dictionary management feature is available for all the languages already supported by our spell-check, i.e., English, French, German, Spanish, Italian, Swedish, Norwegian, Danish, Dutch, and Brazilian.

Reviewer 2: ```suggestion
This dictionary management feature is available for all the languages already supported by our spell-check feature, i.e., English, French, German, Spanish, Italian, Swedish, Norwegian, Danish, Dutch, and Brazilian.
```
